templates/highscore_proto.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Flask Test</title>
+</head>
+<body>
+    	<h1 align="center">Highscore Prototype</h1>
+	<table align="center" border="1" cellspacing="0">
+		<th colspan="3" bghcolor="#000000"><font color="#000000">Highscore</font></th>
```

Comment:
```suggestion
		<th colspan="3"><font color="#000000">Highscore</font></th>
```

Please remove the background colors from the table and table elements

Reply:
Will do the suggested changes